### ISSUE TITLE
[feature]: Adjust Package and Import Ordering in Proto Files for Consistency

### ISSUE THREAD

**Issue by mohamedawnallah** [lightningnetwork/lnd#8301]
**Is your feature request related to a problem? Please describe.**

The package and import ordering in the following Proto files:
- [`walletunlocker.proto`](https://github.com/lightningnetwork/lnd/blob/30348baedcd9e9b157c4cdbf7f0c322fff20cc68/lnrpc/walletunlocker.proto#L3-L5)
- [`walletkit.proto`](https://github.com/lightningnetwork/lnd/blob/30348baedcd9e9b157c4cdbf7f0c322fff20cc68/lnrpc/walletrpc/walletkit.proto#L3-L6)
- [`invoices.proto`](https://github.com/lightningnetwork/lnd/blob/30348baedcd9e9b157c4cdbf7f0c322fff20cc68/lnrpc/invoicesrpc/invoices.proto#L3-L5)
- [`dev.proto`](https://github.com/lightningnetwork/lnd/blob/30348baedcd9e9b157c4cdbf7f0c322fff20cc68/lnrpc/devrpc/dev.proto#L3-L5)
- [`router.proto`](https://github.com/lightningnetwork/lnd/blob/30348baedcd9e9b157c4cdbf7f0c322fff20cc68/lnrpc/routerrpc/router.proto#L3-L5)
- [`lncli.proto`](https://github.com/lightningnetwork/lnd/blob/30348baedcd9e9b157c4cdbf7f0c322fff20cc68/lnrpc/lnclipb/lncli.proto#L3-L5)
- [`peers.proto`](https://github.com/lightningnetwork/lnd/blob/30348baedcd9e9b157c4cdbf7f0c322fff20cc68/lnrpc/peersrpc/peers.proto#L3-L5)

Do not adhere to the recommended structure specified in the style guide. Presently, imports precede the package declaration in these files.

**Describe the solution you'd like**

I would like the package and import ordering within these Proto files to be adjusted according to the prescribed style guide. The desired structure should have the package declaration placed before any import statements.

**Describe alternatives you've considered**

No alternatives have been considered as aligning with the style guide is important for consistency and adherence to best practices.

**Additional context**

Aligning the package and import ordering in these Proto files with the recommended structure will enhance readability, maintainability, and conformity with established Proto file formatting standards as outlined in the style guide here:
https://protobuf.dev/programming-guides/style/#file-structure

**Comment by Roasbeef** [comment]
So minimally, we should have the package declaration _before_ the imports? Ideally we can incorporate a linter here so then the new standard would be enforced by CI.

**Comment by mohamedawnallah** [comment]
@Roasbeef Yeah, it is my first issue with the LND repository :) and I was thinking of the same using using [`protolint` ](https://github.com/yoheimuta/protolint). Please let me know on integrating protolint in the CI I would be happy to help

**Comment by mohamedawnallah** [comment]
Hey @Roasbeef, I've just submitted a PR to integrate the protolinter in our CI so that it enforces the protobuf style guide rules automatically. I'd love to receive any feedback, Thanks!
